### DISCUSSION TITLE
Changing the terminal - rofi keeps using alacritty

### DISCUSSION THREAD

**Discussion by aecardenasfl** [mylinuxforwork/dotfiles#140] [General]
Sorry to open a different post but this is a different issue I'm facing. I changed the default terminal through the "Dot files" App, on the system section from "alacritty" to "kitty" (as this was my terminal emulator of choice before installing ML4W). When I use the keybind SUPER+return it launches kitty just fine. But when I use super+ctrl+return to launch rofi and I want to open for example Neovim, or Yazi it launches those apps in Alacritty. 

I created a custom config file under ~/.config/bashrc/custom and added export TERMINAL=kitty then sourced the ~/.bashrc, rebooted but rofi is still opening everything in Alacritty. I uninstalled Alacritty and it correctly launches the apps in kitty, but if I install it back rofi goes back to launching terminal apps in Alacritty. Not really important but if you maybe have an idea what I'm doing wrong I'd appreciate your feedback.

**Comment by mylinuxforwork** [comment]
I will check and try to reproduce it on my system.

**Comment by hambosto** [comment]
@mylinuxforwork  consider changing the default terminal to kitty. kitty in my opinion is better than alacritty, supports font ligatures too. This is an example of a configuration screenshot that I adapted from alacritty ML4W dotfiles to kitty
![screenshot_15082024_140042](https://github.com/user-attachments/assets/6a7605c8-10b8-44c0-8ed0-d91d3bc96537)
